community/lsof/sources

```diff
@@ -0,0 +1 @@
+https://fossies.org/linux/misc/lsof-4.93.2.tar.gz
```

Comment:
Have a more official source? Fossies is typically used only if the original source is down.